.gitignore

```diff
@@ -2,3 +2,4 @@
 /lila-ws
 /project
 /target
+/.idea
```

Comment:
I don't think we should start adding environment related files

Reply:
It helps preventing someone from accidentally committing IDE configuration. Do you have another trick?

Reply:
Just scrolling through these repositories trying to find any progress on lichess in Docker. I'm interested in running it in Kubernetes.

Here's the way to ignore IDE/OS files in git by the way.
```bash
# git config --global --add core.excludesfile /home/user/.gitignore_global
```